e2e/playwright/shared/api/rules-api.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2005 - 2023 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import {RuleTrigger} from "../../../../projects/aca-folder-rules/src/lib/model/rule.model";
+import {RuleCompositeCondition} from "../../../../projects/aca-folder-rules/src/lib/model/rule-composite-condition.model";
+import {RuleAction} from "../../../../projects/aca-folder-rules/src/lib/model/rule-action.model";
```

Comment:
same as above, incorrect import path, should be using namespace instead of relative paths